### ISSUE TITLE
short/long-press buttons assign 

### ISSUE THREAD

**Issue by jeepy76** [OpenSolo/documentation#7]
How exactly can I edit the  /usr/bin/extSettings.conf  to assign the short/long-press buttons ?

**Comment by hdtechk** [comment]
I use winscp on windows computer to log in and edit files if needed.   There is lots of places to find root password on internet.   They are probably posted here.

**Comment by johngithens** [comment]
I've read /usr/bin/extSettings.conf to better understand the functionality for the Pause button on a Solo Controller with v3.0.0.  That's because I'm not seeing the expected results with a short or long press to change RC7 output to control a servo.  Sometimes goes High, sometimes Low. Appears to be erratic. I may be missing something basic, or maybe the code for the Pause button has not been implemented?

**Comment by lordneeko** [comment]
Pause button code is here.
https://github.com/OpenSolo/artoo/blob/bc83733ae7e7e1ba7a7a0c021d5aaea60a9db3af/src/flightmanager.cpp#L307

https://github.com/OpenSolo/artoo/blob/98e983e1bc7ccdcdb35393aed741b35814a1d017/src/flightmanager.cpp#L949


function implemented in ardupilot-solo:
https://github.com/OpenSolo/ardupilot-solo/blob/6d13352ee124f44cf0dcdc58f2271a64aa1cd6af/ArduCopter/GCS_Mavlink.pde#L1583

Basically it does
set_mode(STOP)
